### DISCUSSION TITLE
Nuget package README.md not showing up in Github packages (except for version specific page)

### DISCUSSION THREAD

**Discussion by imercerwillow** [community/community#64967] [Packages]
### Select Topic Area

Bug

### Body

We have a nuget package with an embedded README.md file. Instead of displaying the readme file, GitHub displays the repository readme and not the nuget's embedded readme.

The embedded README works fine when tested on nuget.org. 

The embedded README works fine when we view a specific version of the nuget file on GitHub.
`https://github.com/<org>/<repo>/pkgs/nuget/<package>/<version>`

But when we view the nuget package's page it displays the repository readme.
`https://github.com/<org>/<repo>/pkgs/nuget/<package>`

Expected the page for a single nuget to display the readme file embedded in the latest revision.

**Comment by arafajs** [comment]
1. Check the formatting of the embedded README file. Make sure that it is a valid Markdown file and that it is formatted correctly.
2. Verify that the embedded README file is included in the NuGet package. Check the nuspec file to make sure that the path to the embedded README file is specified correctly.
3. Reduce the size of the embedded README file. If the embedded README file is too large, you can try reducing the number of words or images in the file.

**Reply by imercerwillow** [reply]
As you can see from the bug report we have already checked 1, 2 and 3. There are no images in the file.

**Reply by georgiosd** [reply]
We're seeing this too - the formatting is all off in the main package page, but looks fine when you visit a particular version page. Any ideas @arafajs ?

**Comment by 9Rune5** [comment]
I don't even see the package's README.md in the version specific repository page.

My .nuspec contains:
`<readme>README.md</readme>`
and the README.md file is indeed in the root directory of the .nupkg archive.

**Comment by joebeernink** [comment]
Anyone from the GitHub Packages team able to convert this to a bug? Really frustrating that we can't see the readme's for the packages.

**Comment by peterfortuin** [comment]
I'm seeing the same problem. I see the README.md that is online in the repo and not the README.md that is included into the NuGet package.

**Reply by JMS-1** [reply]
What is the status of this issue?

**Comment by zhaparoff** [comment]
Same situation here.
Repository root README is displayed instead of package specific README file, although it is specified correctly in the `nuspec` and correct file is located in the root of the `nupkg` archive.
Specific version page does not display any README at all, but displays package description from the nuspec file instead.
Definitely a bug.

**Reply by alexjamesbrown** [reply]
Same here

**Reply by nonsensation** [reply]
It works for other packages, like npm, but not for nuget.

**Comment by iramos-mb** [comment]
This is still a bug, would be great if it loads the package specific readme for documentation.
